core/controllers/base.py

```diff
@@ -80,12 +80,10 @@ def get(self):
         """
 
         _clear_login_cookies(self.response.headers)
-
         url_to_redirect_to = (
             python_utils.convert_to_bytes(
                 self.request.get('redirect_url', '/')))
-        self.redirect(
-            current_user_services.create_logout_url(url_to_redirect_to))
```

Comment:
Are we using this function any more? If not, we should delete it.

Reply:
Done.